YoCode/Output/WebElementBuilder.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Linq;
+
+namespace YoCode
+{
+    static class WebElementBuilder
+    {
+        const string LIST_ELEMENT_TEMPLATE = @"C:\Users\ukekar\source\repos\YoCode\YoCode\Output\ListElementTemplate.html";
```

Comment:
This should not be in a separate file and definitely shouldn't have an absolute path. Using a string resource would solve both problems.